src/odinapi/database/level2db.py

```diff
@@ -326,7 +328,8 @@ def get_measurements(self, products,
                          min_altitude=None, max_altitude=None,
                          min_pressure=None, max_pressure=None,
                          start_time=None, end_time=None, areas=None,
-                         fields=None):
+                         fields=None, min_scanid=None,
+                         document_limit=HARD_LIMIT):
```

Comment:
This changes the meaning of `HARD_LIMIT`, as you may give a higher number. Probably better to default to `math.inf` in the argument and then adjust value with `min(document_limit, HARD_LIMIT)`

Reply:
Remember to update the TODO comments in the file that your changes void.

Reply:
I will remove hard_limit from this part of the code